validation and qualification/README.md

```diff
@@ -2,25 +2,25 @@
 
 ### [(PBPK) Model Evaluation Reports](https://github.com/Open-Systems-Pharmacology/OSP-PBPK-Model-Library)
 
-### [CKD Population Qualification Report](https://github.com/Open-Systems-Pharmacology/Qualification-CKD/blob/main/Qualification/report/report.md)
-
```

Comment:
This will be part of the qualification reports library, so no separate link required.